lib/matplotlib/axes/_axes.py

```diff
@@ -6729,7 +6729,11 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
     def stairs(self, values, edges=None, *,
                orientation='vertical', baseline=0, fill=False, **kwargs):
         """
-        A stepwise constant line or filled plot.
+        A stepwise constant function as a line with bounding edges
+        or a filled plot.
+
+        If an array is passed to *baseline*, while ``fill=False``
```

Comment:
IMHO this belongs to into the *baseline* parameter. It's rather an effect of that parameter than a general information.

Reply:
I would say it's general info, since that option combination produces visually a different object from the default?

Reply:
On it's own it's a very specific info for a special combination of parameters, which I would explain with the parameter. Alternatively, you can make a section on the visual outputs one can create with the function, but that should describe all outputs (open line, closed line, filled polygon).